docs/guides/rookieguide.md

```diff
@@ -1,7 +1,126 @@
 # Poradnik dla rekrutów
 
-## Wstęp
+## *Wstęp*
 
-Poradnik został opracowany z myślą o graczach dopiero rozpoczynających członkostwo w naszej grupie - jego zamiarem jest poprowadzenie was przez rękę przez wszystkie meandry konfigurowania Army i radzenia sobie z wypracowanymi wewnątrz grupy systemami, do których należą między innymi zasady zapisywania się na misję. Poszczególne kroki są dokładnie opisane, te bardziej wymagające popieramy zrzutami ekranów, znajdziecie w nim również podstawowe FAQ dotyczące zagadnień ogólnych. W razie jakichkolwiek problemów ze zrozumieniem treści poradnika kontaktujcie się ze swoimi opiekunami Rekruterami, lub uderzajcie do jego twórcy na Discordzie - @Nakimeteles.
+Poradnik został opracowany z myślą o graczach dopiero rozpoczynających członkostwo w naszej grupie - jego zamiarem jest poprowadzenie was przez rękę przez wszystkie meandry konfigurowania Army i radzenia sobie z wypracowanymi wewnątrz grupy systemami, do których należą między innymi zasady zapisywania się na misję. Poszczególne kroki są dokładnie opisane, te bardziej wymagające popieramy zrzutami ekranów, znajdziecie w nim również podstawowe FAQ dotyczące zagadnień ogólnych. W razie jakichkolwiek problemów ze zrozumieniem treści poradnika kontaktujcie się ze swoimi opiekunami Rekruterami.
 
-[Przejdź do poradnika dla rekrutów](https://docs.google.com/document/d/1nOPqnTeu9Flph8oBtN2FFMQ5dUvwFet_Y80uH1ckp1s/edit?usp=sharing)
+## *Discord Arma Forces*
+
+### 1. Wchodzimy na Discord Arma Forces.
+
+
+### 2. Wchodzimy na kanał #rekrut_game_annoucments
+
+
+
+Obrazek orientacyjny:
+
+
+![Obrazek orientacyjny](../_data/guides/rookie/rookie1.png)
+
+
+### 3. Na tym kanale znajdują się zapisy na misje ze slotami dla rekrutów. Zapisy działają na zasadzie odklikania reakcji, zasady listuje rekruter wysyłający ogłoszenie. 
+
+Przykładowe zapisy dla rekrutów:
+
+![Przykładowe zapisy](../_data/guides/rookie/rookie2.png)
+
+### Misje zawsze odbywają się o 20 chyba, że zostanie inaczej powiedziane.
+
+
+## Pobieranie modów
+
+### 1. Pobieranie modów.
+
+Klikamy na podany w zapisach link do modów, co przeniesie nas to na stronę z modlistami. Modlisty składają się z dwóch części: modów wymaganych, które potrzebne są do wejścia na serwer gdy dana modlista jest załadowana oraz modów opcjonalnych, które nie są potrzebne by rozegrać misje, lecz zawierają różnego rodzaju mody na lepsze dźwięki, dodatkowe ustawienia graficzne i tym podobne. Jesteście samemu w stanie zaznaczyć, które mody opcjonalne chcecie pobrać wraz z modlistą. Oczywiście możecie je, także w pełni zignorować, ale unikajcie dodawania ich wszystkich ponieważ część z nich się wyklucza lub w połączeniu może powodować różne błędy. Większość modów posiada krótki opis, kiedy na nie najedziecie, z którego możecie skorzystać w razie wątpliwości. Kiedy już dokonaliście wyboru klikamy przycisk "Pobierz/Download" i czekamy aż modlista się pobierze.
```

Comment:
```suggestion
Klikamy na podany w zapisach link do modów, co przeniesie nas to na stronę z modlistami. Modlisty składają się z dwóch części: modów wymaganych, które potrzebne są do wejścia na serwer gdy dana modlista jest załadowana oraz modów opcjonalnych, które nie są potrzebne by rozegrać misje, lecz zawierają różnego rodzaju mody na lepsze dźwięki, dodatkowe ustawienia graficzne i tym podobne. Jesteście sami w stanie zaznaczyć, które mody opcjonalne chcecie pobrać wraz z modlistą. Oczywiście możecie je w pełni zignorować, ale unikajcie dodawania ich wszystkich, ponieważ część z nich się wyklucza lub w połączeniu może powodować różne błędy. Większość modów posiada krótki opis, kiedy na nie najedziecie, z którego możecie skorzystać w razie wątpliwości. Kiedy już dokonaliście wyboru klikamy przycisk "Pobierz/Download" i czekamy, aż modlista się pobierze.
```

Reply:
I saw that you implemented requested changes, but unfortunately there is still a linguistic mistake. It should be "Jesteś w stanie sam zaznaczyć", not "Jesteś w stanie samemu zaznaczyć".